addons/sys_core/initSettings.sqf

```diff
@@ -47,3 +47,59 @@
     false,
     {EGVAR(sys_radio,disableDesyncHint) = _this}
 ] call CBA_Settings_fnc_init;
+
+// Difficulty settings
+// Interference
+[
+    "ACRE_INTERFERENCE",
```

Comment:
Those variables should probably be macroized to be consistent with others, they are not public API. Probably best if they stick to being `acre_sys_core_<name>`.